### DISCUSSION TITLE
HA Android app: high RAM usage of the Home Assistant app since installing Frigate

### DISCUSSION THREAD

**Discussion by ZeroPing01** [blakeblackshear/frigate#13401] [General]
Hi,
Device/OS : android 14
HA app : 2024.7.3-full
Frigate with proxy addon
Frigate 0.14

I upgraded to frigate 0.14 and I’ve experiencing high memory usage with the Home Assistant app on my Android phone since I installed Frigate.

![image](https://github.com/user-attachments/assets/9e43fa01-1111-41f7-b33b-98c43113fc39)

What could be the technical explanation for this?
Are there that many data being loaded on the frontend (client-side)?

I often need to close and reopen the app; otherwise, it becomes slow or unresponsive.

Have you noticed this as well?
What can I do to fix it?

Thx

**Comment by NickM-27** [comment]
I don't see any reason for this. I have frigate installed as a PWA and it doesn't use anything like that

![Screenshot_20240828_062411_Settings.jpg](https://github.com/user-attachments/assets/623db9ab-f613-43b9-9508-82b67ebf9a91)

**Reply by ZeroPing01** [reply]
I didn't have these problems with Frigate 0.13, I don't understand. 
Strange, I don't know what to do.
